### PR TITLE
Added win build in release (experimental)

### PR DESCRIPTION
**Pull request by Yshayy**
**[Is this a bugfix/feature/doc-improvement?]**
## This is a

- [ ] Bug fix
- [x] Feature
- [ ] Doc improvement

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

- [x] I have used Preevy for a while and am familiar with the function it providers.

If this is a bug fix or feature:

- [ ] I tested the proposed change on my cloud provider: **Please specify which provider**
- [ ] I tested the proposed change on a local Kubernetes cluster.

### CODE REVIEW

File: packages/core/src/compose/model.ts [hunk 1]
```diff
@@ -69,6 +69,8 @@ const volumeSkipList = [
   /^\/$/,
 ]
 
+const toPosix = (x:string) => x.split(path.sep).join(path.posix.sep)
```
Comment (royra) [inline packages/core/src/compose/model.ts:72]: what is the difference between this and 
```typescript
x.replaceAll(path.sep, path.posix.sep)
```

**Review by royra — APPROVED** [review]
see comments

File: build_utils/eslintrc.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ module.exports = {
   plugins: ['import', 'jest', '@typescript-eslint'],
   rules: {
     'no-void': ['warn', { allowAsStatement: true }],
-    'linebreak-style': ['warn', isWindows ? 'windows' : 'unix'],
+    'linebreak-style': ['warn', 'unix'],
```
[the remark this answers is not in the document]
Comment (Yshayy) [reply build_utils/eslintrc.js:37]: When I run locally, it created a lot of noise and added files to the commit itself.

File: packages/core/src/compose/model.ts (hunk 1 above)
Comment (Yshayy) [reply packages/core/src/compose/model.ts:72]: I assume there's no difference

**Pull request merged**